lib/Hakyll/Web/Pandoc.hs

```diff
@@ -138,19 +166,7 @@ pandocCompilerWithTransformM :: ReaderOptions -> WriterOptions
                     -> (Pandoc -> Compiler Pandoc)
                     -> Compiler (Item String)
 pandocCompilerWithTransformM ropt wopt f = 
-    getResourceBody >>= applyPandocWith ropt wopt f
-
-
---------------------------------------------------------------------------------
--- | Similar to pandocCompilerWithTransformM, but takes an `Item String` as an
--- additional argument. Useful if you want to do transformations before doing a
--- custom Pandoc transformation, e.g. using custom templating functions.
-applyPandocWith :: ReaderOptions -> WriterOptions
-                    -> (Pandoc -> Compiler Pandoc)
-                    -> Item String
-                    -> Compiler (Item String)
-applyPandocWith ropt wopt f i = 
-    writePandocWith wopt <$> (traverse f =<< readPandocWith ropt i) 
+    getResourceBody >>= renderPandocWithTransformM ropt wopt f
```

Comment:
Moved function body up next to the other `renderPandoc` functions and changed the name where it's used here.